wallet/src/Pos/Wallet/Web/Methods/Logic.hs

```diff
@@ -156,11 +162,15 @@ getAccountsIncludeUnready ws mps includeUnready mCAddr = do
       void $ maybeThrow (noWallet cAddr) $
         getWalletMetaIncludeUnready ws includeUnready cAddr
     let accIds = maybe (getAccountIds ws) (getWalletAccountIds ws) mCAddr
+    logInfo $ sformat ("getAccountsIncludeUnready: Computing balance for " % shown) accIds
     let groupedAccIds = fmap reverse $ HM.fromListWith mappend $
                         accIds <&> \acc -> (aiWId acc, [acc])
-    concatForM (HM.toList groupedAccIds) $ \(wid, walAccIds) -> do
-      accMod <- txMempoolToModifier ws mps =<< findKey wid
-      mapM (getAccountMod ws accMod) walAccIds
+    logInfo $ sformat ("getAccountsIncludeUnready: Grouped AccountId(s) " % shown) groupedAccIds
```

Comment:
The other PR includes a bit more here. Do we not want to include the same in this version?
```
+    logInfo $ sformat ("getAccountsIncludeUnready: " % stext) (renderUtxo    . snd $ balAndUtxo)
+    logInfo $ sformat ("getAccountsIncludeUnready: " % stext) (renderBalance . fst $ balAndUtxo)
```